packages/react/src/blocks/Image.tsx

```diff
@@ -261,6 +263,13 @@ class ImageComponent extends React.Component<any, { imageLoaded: boolean; load:
     return this.props.image || this.props.src;
   }
 
+  get fetchPriorityProp() {
+    const propNameToUse = isNewReact ? 'fetchPriority' : 'fetchpriority';
+    return {
+      [propNameToUse]: this.props.highPriority ? 'high' : 'auto',
```

Comment:
The previous logic for this value was 
`const eagerLoad = isPixel || this.props.highPriority;`

Do we need to add `isPixel` check here?